src/rules.jl

```diff
@@ -518,6 +530,45 @@ function lrp!(Rᵏ, rule::GeneralizedGammaRule, layer, modified_layers, aᵏ, R
     @. Rᵏ = aᵏ⁺ * (cˡ⁺ + cʳ⁻) + aᵏ⁻ * (cˡ⁻ + cʳ⁺)
 end
 
+"""
+    LayerNormRule(affine_rule=ZeroRule())
+
+LayerNormRule rule.
+
+# Optional arguments
+- `affine_rule`: Rule applied to affine transformation, defaults to `ZeroRule()`.
+
+# References
+- 
+"""
+struct LayerNormRule{T <: AbstractLRPRule} <: AbstractLRPRule
+    affine_rule::T
+end
```

Comment:
Do we need this wrapper around other rules?
Maybe defining `lrp!(Rᵏ, rule, layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)` (without the limitation on `rule::LayerNormRule`) and using generic rules is enough?

Reply:
RelevancePropagation is quite unique in its ability to assign rules to both layer and rule types, so a distinction between
```Julia
lrp!(Rᵏ, rule::LayerNormRule(GammaRule()), layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
```
and
```Julia
lrp!(Rᵏ, rule::GammaRule(), layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
```
might not be necessary.